### PR TITLE
Release 1.5.0

### PR DESCRIPTION
**Pull request by clechot**

### CODE REVIEW

**Comment by clechot** [comment]
Do we want to mention https://github.com/Esri/pyprt/pull/130 in the CHANGELOG?

**Review by mistafunk — APPROVED** [review]
LGTM.

File: README.md [hunk 1]
```diff
@@ -158,9 +158,12 @@ Note: We only support Docker on Linux and Windows. On Windows, Docker needs to b
 #### Build Wheels
 
 1. Open a shell in the PyPRT git root
+1. Create the base image for the desired build toolchain (adapt to your desired Python version):
+   * Linux: `docker build --rm -f envs/centos7/base/Dockerfile -t pyprt-base:centos7-py3.6 --build-arg PY_VER=3.6 --build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g) .`
+   * Windows: `docker build --rm -f envs\windows\base\Dockerfile -t pyprt-base:windows-py3.6 --build-arg PY_VER=3.6 .`
```
Comment (mistafunk) [inline README.md:163]: For wheels on Windows, we use a different base image dockerfile: `Dockerfile-py` (prebuilt windows images provided by the python community)

**Review by mistafunk — CHANGES_REQUESTED** [review]
* Looks good overall, see one inline comment...
* In the readme, we should remove "We only support Docker on Linux and Windows." in the docker section. Does not make any sense anymore after removing macOS support.
* Consider running our PyPRT-Docker pipeline before merging
* There are merge conflicts with develop, consider rebasing...

**Review by mistafunk — APPROVED** [review]
Nice work, feel free to merge.

**Pull request merged**